strings/base_delegate.h

```diff
@@ -6,39 +6,63 @@ namespace winrt::impl
 #pragma warning(disable:4458) // declaration hides class member (okay because we do not use named members of base class)
 #endif
 
-    template <typename T, typename H>
-    struct implements_delegate : abi_t<T>, H, update_module_lock
+    template<bool try_marshaling> struct implements_delegate_base
     {
-        implements_delegate(H&& handler) : H(std::forward<H>(handler))
+        WINRT_IMPL_NOINLINE uint32_t inc_ref() noexcept
```

Comment:
The cppwinrt library avoids abbreviations so this would be `increment_reference`.